doc/rst/platforms/infiniband.rst

```diff
@@ -5,8 +5,11 @@ Using Chapel with InfiniBand
 ============================
 
 This document describes how to run Chapel across multiple machines in an
-InfiniBand cluster. The :ref:`readme-multilocale` describes general
-information about running Chapel in a multilocale configuration.
+InfiniBand cluster using GASNet-based communication.
+:ref:`readme-multilocale` describes general information about running
+Chapel in a multilocale configuration.  The ofi communication layer can
+also be used on InfiniBand-based systems; see :ref:`readme-libfabric`
+for more information.
```

Comment:
I would prefer that we don't recommend or point ordinary users to the ofi layer until we have regular nightly testing (and maybe a sense of the performance here for gasnet-ibv vs ofi-verbs.)

i.e. I'm totally fine with talking about ibv in the ofi document, but I don't want to point ordinary users to it just yet.

Reply:
Are you saying you don't think any of the general docs should have references to the ofi one until we are at least running the hellos?  Or something more specific?

Reply:
I don't think we should have general docs referring to the ofi until we're ready to recommend it to users (and I'm using regular testing and performance results as a proxy for that)

Reply:
@ronawho and I had a chat and I'm going to remove the change to this file, but the rest will stay how they are (pending further feedback, of course!).